Lib/fontTools/ttLib/ttFont.py

```diff
@@ -570,8 +582,9 @@ def getReverseGlyphMap(self, rebuild=False):
 	def _buildReverseGlyphOrderDict(self):
 		self._reverseGlyphOrderDict = d = {}
 		glyphOrder = self.getGlyphOrder()
-		for glyphID in range(len(glyphOrder)):
-			d[glyphOrder[glyphID]] = glyphID
+		for glyphID,glyphName in enumerate(self.getGlyphOrder()):
```

Comment:
You should use `glyphOrder` here instead of calling `self.getGlyphOrder` again, or remove the `glyphOrder` variable.